introduction/telemetry.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Telemetry"
+description: "Using `tracing` or `log` with shuttle apps."
+---
+
+## Tracing
+
+When you build an app with the `#[shuttle_service::main]` macro, a global subscriber will be created and installed
+[behind the scenes](https://github.com/shuttle-hq/shuttle/blob/001dbcfcf47066713598f00a98f4972de450ff3b/codegen/src/main/mod.rs#L237-L245).
+This means if you can skip this step when implementing [tracing](https://docs.rs/tracing/latest/tracing/) in your application, 
+all you have to do is add `tracing` as a dependency in your `Cargo.toml`, and you're good to go!
```

Comment:
It might be usefull to include a minimal example of one log / tracing call

Reply:
Yeah, that's a good idea. Thanks!